### DISCUSSION TITLE
Getting a 404 error when I shouldn't be (UE Blender addon)

### DISCUSSION THREAD

**Discussion by wonderland78** [community/community#57203] [New to GitHub]
On this page: https://www.unrealengine.com/en-US/blog/download-our-new-blender-addons

At this paragraph:
To download our new, free addons, [log in to your Epic Games account and connect it with your GitHub account](https://www.unrealengine.com/en-US/ue4-on-github) then access the addons and documentation [here](https://github.com/epicgames/blendertools).

When I click the link it gives a 404 error. You need to connect your Unreal account to Github, which as you can see I've done here:

![image](https://github.com/community/community/assets/107247634/7a362d49-700e-4494-927c-49bebea75f6f)

And other people have told me they can access the link so it's not down.

**Comment by queenofcorgis** [comment]
Hi there - have you reached out to Unreal Engine support about this as well?
